web/packages/teleport/src/config.ts

```diff
@@ -1105,6 +1112,17 @@ const cfg = {
     });
   },
 
+  getDeleteIntegrationUrlV2(req: IntegrationDeleteRequest) {
+    const path = cfg.api.integration.deleteV2
+      .replace(':clusterId', req.clusterId)
+      .replace(':name', req.name)
+      .replace(
+        ':associatedresources',
+        req.associatedResources ? 'true' : 'false'
+      );
+    return generatePath(path, req);
```

Comment:
Should this be
```suggestion
    return generatePath(cfg.api.integration.deleteV2, {
      clusterId: req.clusterId,
      name: req.name,
      associatedresources: req.associatedResources ? 'true' : undefined,
    });
```

I think `undefined` could work instead of `'false'` as it doesn't need to be set

Reply:
the `generatePath` didn't work in scenarios where we have a dynamic path and a query param next to each other eg: `some/path/:dynamicName?queryParam1=queryParamValue` the `generatePath` i think sees `:dynamicName?` as part of like an optional query param and removes it.
 
we do the same manual replacing of paths for the unified resources with query params. should i leave a note?